smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftSettings.kt

```diff
@@ -67,8 +65,6 @@ class SwiftSettings(
     val copyrightNotice: String,
     val visibility: String,
     val internalClient: Boolean,
-    val generatePackageManifest: Boolean,
-    val generateDependencyJSON: Boolean,
```

Comment:
These two `SwiftSettings` fields are no longer used since the package manifest is now always generated, and the `Dependencies.json` file generation has been moved to SDK.